package.json

```diff
@@ -16,6 +16,8 @@
     "reason-react": "^0.9.1"
   },
   "optionalDependencies": {
```

Comment:
does this work? optionalDependencies?

should these be peerDependencies and then we install them in the builder explicitly so we know what we're using?

Reply:
I think `optional` is correct here -- `peer` means that the library *needs* the dependency, but it's not providing itself. `optional` means you can choose to use it or not